.github/workflows/test.yml

```diff
@@ -6,25 +6,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["pypy-3.7", "3.9"]
+        python-version: ["pypy-3.9", "3.11"]
```

Comment:
Shouldn't it be pypy 3.8 (3.7 was dropped but the lower bound seems to be 3.8 not 3.9) ?

Reply:
These are UPPER bounds, not lower bounds.

Reply:
I thought it was lower bound (3.7) and upper bound (3.9). Wouldn't we use 3.9 only if it was upper bounds ?